### DISCUSSION TITLE
Global Actions

### DISCUSSION THREAD

**Discussion by kevin-cauchi-rightship** [community/community#114644] [Actions]
### Select Topic Area

Product Feedback

### Body

It would be amazing if we could have global actions across the repos. It is extremely painful to do this change across hundreds of repos manually or API. Any plans for this?

**Comment by maksymhu** [comment]
To automate global actions across multiple repositories, such as making changes across hundreds of repos simultaneously, you can consider using tools like GitHub Actions or similar automation tools. By creating workflows and scripts, you can streamline and automate tasks across multiple repositories, saving time and effort. Would you like more information on how to set up automation for global actions across repositories?

**Reply by kevin-cauchi-rightship** [reply]
So basically you're telling me to create and action to sync actions?

**Reply by maksymhu** [reply]
Just similar to this.

**Comment by amardotai** [comment]
Ultimately, the most suitable approach will depend on factors such as the scale of your repositories, your organization's existing infrastructure and processes, and the specific requirements of the actions you wish to perform. By leveraging a combination of automation tools, scripting, and best practices, you can streamline the process of managing actions across multiple repositories.
